manifests/l/LibRetro/RetroArch/Nightly/Nightly/LibRetro.RetroArch.Nightly.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: LibRetro.RetroArch.Nightly
+PackageVersion: Nightly

```

```suggestion
PackageVersion: 1.10.3.0
```
This must match Control Panel.